Doc/c-api/mapping.rst

```diff
@@ -9,7 +9,8 @@ Mapping Protocol
 .. c:function:: int PyMapping_Check(PyObject *o)
 
    Return ``1`` if the object provides mapping protocol, and ``0`` otherwise.  This
-   function always succeeds.
+   function always succeeds. This is not exactly equivalent to ``collections.abc.Mapping``.
```

Comment:
Use double space after sentence-ending period. The line is too long, wrap it at column 79. Make collections.abc.Mapping a reference.